AmplifyTools/amplify-tools.sh

```diff
@@ -5,15 +5,23 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-set -e
 
 if ! which node >/dev/null; then
   echo "warning: Node is not installed. Visit https://nodejs.org/en/download/ to install it"
   exit 1
 fi
 
+# Check for NVM and make sure it's initialized
+NVM_PATH="${HOME}/.nvm/nvm.sh"
+if [ -f "${NVM_PATH}" ]; then
+  echo "NVM found, initializing it..."
+  source "${NVM_PATH}"
+fi
+
 export PATH=$PATH:$(npm bin -g)
 
+set -e
+
```

Comment:
Sorry, I should have caught this before....

`set -e` should probably come before `export PATH=$PATH:$(npm bin -g)` in case `npm bin` fails.